README.md

```diff
@@ -476,23 +506,30 @@ to the probe's clock. This can then be included in your logging as a
 breadcrumb for finding a specific event in a trace. That might look
 something like this:
 
-```rust
-let instant = probe.now();
-log::info!("Producer now {:?}", instant);
+```c
+const modality_probe_instant now = modality_probe_now(g_producer_probe);
+syslog(
+        LOG_INFO,
+        "Producer now "
+        "(id: %" PRIu32 ", epoch: %" PRIu16 ", ticks: %" PRIu16 ", event_count: %" PRIu32 ")\n",
+        now.clock.id,
+        now.clock.epoch,
+        now.clock.ticks,
+        now.event_count);
 ```
 
 This will place a Modality causal-coordinate into your log message, so
 that later in offline processing any given log message can be
 correlated with a specific location in the Modality probe's logical
 timeline. You can now stitch together the causal history of your
-typical device logging along side Modality's events & expectations.
+typical device logging along side Modality's events and expectations.
 
-## Running the tests
+## Running the Tests
 
-To run the Rust unit & property-based test suites you need to run:
+To run the C API tests use:
 
 ```shell
-$ cargo test --features std
+$ cargo test
```

Comment:
Why removed?

Reply:
:+1: Another inheritance from moving over the example from the C-API. I'm not sure why it was removed, and `cargo test --features std` works for me, so reverting back to:
`$ cargo test --features std`